pymeasure/instruments/etslindgren/m7006_001.py

```diff
@@ -0,0 +1,154 @@
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import logging
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class M7006_001(Instrument):
+    """Position control card of EMCenter modular system.
+    Allows for controll and feedback of the positioner card.
+    Comunication happes transparently through the EMCenter mainframe.
+    devices share the resource name of the mainframe, and
+    are identified by the slot number and the device number
+    (if applicable to the modual).
+    """
+
+    def __init__(self, resource_name, slot=1, device="A", **kwargs):
+
+        kwargs.setdefault("write_termination", "\n")
+        super().__init__(resource_name, "test", **kwargs)
+        self._slot = slot
+        self._device = device
+
+    def write(self, command):
+        super().write(f"{self._slot}{self._device}{command}")
+
+    def values(self, command, **kwargs):
+        return super().values(f"{self._slot}{self._device}{command}", **kwargs)
+
+    acceleration = Instrument.control(
+        "ACC?",
+        "ACC %g",
+        """Acceleration of the device in seconds. Settable 0.1 and 30,
+        and gettable.
+        """,
+        values=(0.1, 30),
+        validator=strict_range,
+    )
+
+    speed = Instrument.control(
+        "SPEED?",
+        "SPEED %g",
+        """Speed of the device, as a percentage of the max speed.
+        Settable 0.0 to 100.0, and gettable.""",
+        values=(0.0, 100.0),
+        validator=strict_range,
+    )
+
+    aux1 = Instrument.control(
+        "AUX1?",
+        "AUX1 %g",
+        """Auxiliary input 1, Settable to ON or OFF, and gettable.""",
+        values=("ON", "OFF"),
+        validator=strict_discrete_set,
+    )
+
+    aux2 = Instrument.control(
+        "AUX2?",
+        "AUX2 %g",
+        """Auxiliary input 2, Settable to ON or OFF, and gettable.""",
+        values=("ON", "OFF"),
+        validator=strict_discrete_set,
+    )
+
+    rotation = Instrument.setting(
+        "%g",
+        """Rotate the device in the given direction. CW or CCW.""",
+        values=("CW", "CCW"),
+        validator=strict_discrete_set,
+    )
+
+    movement = Instrument.setting(
+        "%g",
+        """Move the tower up or down.""",
+        values={"up": "UP", "down": "DN"},
+        map_values=True,
+    )
```

Comment:
Do these move the device by one increment/step or so? If yes, I think methods `up()/down()/cw()/ccw()` (or `move(dist=+1)/rotate(angle=1)` could be more appropriate -- with the added benefit that you could pass a step size/distance to those methods, too.